docs/messaging-service/email-config.md

```diff
@@ -0,0 +1,32 @@
+# Email gateway configuration
+
+The Messaging service supports 3 ways of sending email messages out of the box:
```

Comment:
...out of the box, by integrating with:
- The OneWelcome email gateway
- A SMTP server
- A custom API

Reply:
fixed